addons/project/models/project.py

```diff
@@ -0,0 +1,816 @@
+# -*- coding: utf-8 -*-
```

Comment:
I agree with @dreispt! It would be so much cleaner and easier if there was one Python file for ever model.